stacks-signer/src/signerdb.rs

```diff
@@ -826,6 +826,16 @@ impl SignerDb {
         try_deserialize(result)
     }
 
+    /// Return the count of globally signed blocks in a tenure (identified by its consensus hash)
+    pub fn get_globally_signed_block_count_in_tenure(
+        &self,
+        tenure: &ConsensusHash,
+    ) -> Result<i64, DBError> {
+        let query = "SELECT COUNT(*) FROM blocks WHERE consensus_hash = ?1 AND state = ?2";
```

Comment:
Please don't use `COUNT(*)` since it requires a table scan.  Can you instead do something like query the difference between the highest and lowest block heights in the tenure?  This would only require a table search if these fields are indexed.

Reply:
Good point.

Reply:
I think you might need a `+1` in there, like, `SELECT (MAX(stacks_height) - MIN(stacks_height) + 1) AS block_count FROM blocks WHERE consensus_hash = ?1 AND state = ?2`